terraform/api_gateway.tf.json.template.py

```diff
@@ -105,6 +105,18 @@ def for_domain(cls, domain):
                     "certificate_arn": "${aws_acm_certificate_validation.%s_%i.certificate_arn}" % (lambda_.name, i)
                 } for i, domain in enumerate(lambda_.domains)
             },
+            "aws_api_gateway_method_settings": {
+                f"{lambda_.name}_{i}": {
+                    "rest_api_id": "${module.chalice_%s.RestAPIId}" % lambda_.name,
+                    "stage_name": "${module.chalice_%s.stage_name}" % lambda_.name,
+                    "method_path": "*/*",
```

Comment:
Why not just `*`? Is the pattern behavior documented somewhere?

Reply:
[Terraform `method_path`](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/api_gateway_method_settings#method_path) documents the usage of `*/*`.

Reply:
```suggestion
                    "method_path": "*/*",  # every URL path, every HTTP method
```